packages/runtime/container-runtime/src/test/scheduleManager.spec.ts

```diff
@@ -32,7 +32,7 @@ describe("ScheduleManager", () => {
 				deltaManager,
 				emitter,
 				() => testClientId,
-				createDebugLogger({ namespace: "fluid:testScheduleManager" }),
+				createChildLogger({ namespace: "fluid:testScheduleManager" }),
```

Comment:
If we move forward with these replacements, does it make sense to provide a namespace? Before, we could enable console logging while running the tests and potentially find the messages in the output, but a ChildLogger on its own will not forward the events anywhere, so I don't see how we would ever get access to any logs written through it. We're effectively passing a null logger, right?

Reply:
I just did find replace, we could remove it, but it doesn't hurt anything in test. This is effectively a null logger.

Its true that this remove the capability to see logs for these specific unit test, but i'm not aware of that ever being used, in general i think people would just debug directly, rather that look at logs. all e2e test will continue to emit debug logs, as they use the loader.